db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/cleanup.rb

```diff
@@ -1,26 +1,41 @@
 INVENTORY_FILE = 'inventory.yaml'.freeze
 RHEL_SUBSCRIBE_INVENTORY = 'rhel_subscribe_inventory.yaml'.freeze
 
+def find_vm_by_tag(tag)
+  tag = "/managed/miq_openshift_deploy/#{tag}_#{$evm.root['automation_task'][:id]}"
+  $evm.vmdb(:vm).find_tagged_with(:any => tag, :ns => "*")
+end
+
 def cleanup
-  $evm.log(:info, "********************** #{$evm.root['ae_state']} ***************************")
-  begin
-    if File.exist?(INVENTORY_FILE)
-      $evm.log(:info, "deleting #{INVENTORY_FILE}")
-      system "sudo rm #{INVENTORY_FILE}"
-    end
-    if File.exist?(RHEL_SUBSCRIBE_INVENTORY)
-      $evm.log(:info, "deleting #{RHEL_SUBSCRIBE_INVENTORY}")
-      system "sudo rm #{RHEL_SUBSCRIBE_INVENTORY}"
+  $evm.log(:info, "********************** Cleanup ***************************")
+  if File.exist?(INVENTORY_FILE)
+    $evm.log(:info, "deleting #{INVENTORY_FILE}")
+    system "sudo rm #{INVENTORY_FILE}"
```

Comment:
we need this sudo away, it's sudo on MIQ apliance, for me it hangs on sudo prompt